californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
These are the same exports as used in the californium-osgi bundle. It doesn't introduce a change in the relation between californium-osgi and californium-core.

Basically it is the pre-requisite to make californium-osgi a better OSGi citizen. Currently it includes californium-core and element-connector, which is not a good OSGi practice. If both bundles provide proper OSGi meta-data, they don't need to be wrapped anymore but can be simply dependent, which means you can simple have all three bundles in the runtime.